### DISCUSSION TITLE
request script for Glances

### DISCUSSION THREAD

**Discussion by Sitherious** [tteck/Proxmox#2858] [Ideas]
i love your scripts, currently using allot of them, but the one im using the most is homepage, and homepage offers support for glances, please add a script for glances.

**Comment by tteck** [comment]
To install Glances on an existing LXC

```
wget -O- https://bit.ly/glances | /bin/bash
cat <<EOF >/etc/systemd/system/glances.service
[Unit]
Description=Glances - An eye on your system
After=network.target

[Service]
Type=simple
ExecStart=/usr/local/bin/glances -w
TimeoutStopSec=20
Restart=on-failure

[Install]
WantedBy=multi-user.target
EOF
systemctl enable -q --now glances.service
```
Glances Interface: IP:61208

**Reply by Sitherious** [reply]
ty

**Reply by tteck** [reply]
Did that work for you?

**Reply by Sitherious** [reply]
i will test it once i get home later tonight

**Comment by Sitherious** [comment]
works, tho im getting an error in homepage but thats prob my fault. other then that it works like a charm thanks

**Comment by aroundmyroom** [comment]
tried it on a fresh unpriviliged lxc container with debian bookworm
but although the system tells me all is ok

the usr/local/bin part is skipped as I see this error:

Install dependencies
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.

etc..

**Reply by tteck** [reply]
You are installing it on a LXC not created by a script in this repository.
You can run `rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED` to resolve your issue.

**Reply by aroundmyroom** [reply]
thanks, that solved the issue

**Comment by TDSnet** [comment]
hi,
Thanks for your terrific job! Many warnings advise against deleting this file (EXTERNALLY-MANAGED) with regard to future maintenance via "Apt". They recommend using Pipx to install Glances. What do you think ? Is the risk real?
regards
